### PR TITLE
Sils/docs

### PR DESCRIPTION
**Pull request by sils1297**

### CODE REVIEW

**Comment by Makman2** [comment]
ack 14d2d19

**Comment by Makman2** [comment]
ack 1809c21

**Comment by Makman2** [comment]
ack a8822a9

File: README.rst [hunk 1]
```diff
@@ -1,3 +1,5 @@
+You might also want to visit `our website<http://coala-analyzer.org/>`_.
```
Comment (Makman2) [inline README.rst]: you forgot a space, the link is broken

**Comment by Makman2** [comment]
unack ee74f54

**Comment by Makman2** [comment]
ack 622090e

**Comment by sils1297** [comment]
@rultor merge

**Pull request merged**